src/redux/board.slice.ts

```diff
@@ -22,11 +22,10 @@ const convertBoardToInnerType = (ob: OuterBoard): innerBoard => {
 /**
  * getBoardsThunk call the function to hit the API to fetch Board list data.
  */
-export const getBoardsThunk = createAsyncThunk(
+export const getBoardThunk = createAsyncThunk(
   "board/getBoardsThunk",
```

Comment:
Probably should change "board/getBoardsThunk" to "board/getBoardThunk" as well, right?

Reply:
Yes you are right, I fixed it. Thanks!